src/Elliptic/Systems/Elasticity/BoundaryConditions/LaserBeam.cpp

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Elliptic/Systems/Elasticity/BoundaryConditions/LaserBeam.hpp"
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/DotProduct.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Elliptic/BoundaryConditions/BoundaryCondition.hpp"
```

Comment:
is `BoundaryCondition.hpp` actually used in the cpp? It seems to only be implementing the `Impl` class.